### PR TITLE
for #20: start 3rd onboarding panel; update images

### PR DESCRIPTION
**Pull request by groovecoder**
Not ready to merge yet, just getting started.

### CODE REVIEW

**Comment by groovecoder** [comment]
Okay, this should be ready to check.

File: webextension/js/popup.js [hunk 1]
```diff
@@ -154,8 +157,29 @@ Logic.registerPanel(P_ONBOARDING_2, {
...
+    });
+  },
+
+  // This method is called when the panel is shown.
+  prepare() {
+    return Promise.resolve(null);
+  },
+});
+
+// P_ONBOARDING_2: Second page for Onboarding.
```
Comment (pdehaan) [inline webextension/js/popup.js]: nit: update comment

File: webextension/popup.html [hunk 2]
```diff
@@ -8,21 +8,30 @@
...
+      Use containers to organize tasks, manage accounts, and keep your focus where you want it.
+      <br/>
+      <a href="#">Learn more</a>
+    </p>
+    <a href="#" id="onboarding-start-button" class="button expanded onboarding-button">Get Started</a>
   </div>
 
 
   <div class="panel onboarding onboarding-panel-2 hide" id="onboarding-panel-2">
     <img alt="How Containers Work" src="/img/onboarding-2.png" />
```
Comment (pdehaan) [inline webextension/popup.html]: Does this need `class="onboarding-img"` as well?

File: webextension/popup.html [hunk 3]
```diff
@@ -8,21 +8,30 @@
...
     <h3>Put containers to work for you.</h3>
     <p>Features like color-coding and separate container tabs help you find things easily, focus your attention, and minimize distractions.</p>
-    <a href="#" id="onboarding-done-button" class="button expanded spacing">Done</a>
+    <a href="#" id="onboarding-next-button" class="button expanded spacing">Next</a>
   </div>
 
+  <div class="panel onboarding onboarding-panel-3 hide" id="onboarding-panel-3">
+    <img alt="How Containers Work" src="/img/onboarding-3.png" />
+    <h3>A place for everything, and everything in its place.</h3>
```
Comment (pdehaan) [inline webextension/popup.html]: Do we need to add `class="onboarding-img"` and `class="onboarding-title"` here as well (similar to panel-1 above)?

**Comment by groovecoder** [comment]
Thanks for the review @pdehaan - addressed all comments.

File: webextension/popup.html [hunk 4]
```diff
@@ -8,21 +8,30 @@
...
+    <img class="onboarding-img" alt="How Containers Work" src="/img/onboarding-2.png" />
+    <h3 class="onboarding-title">Put containers to work for you.</h3>
     <p>Features like color-coding and separate container tabs help you find things easily, focus your attention, and minimize distractions.</p>
-    <a href="#" id="onboarding-done-button" class="button expanded spacing">Done</a>
+    <a href="#" id="onboarding-next-button" class="button expanded spacing">Next</a>
   </div>
 
+  <div class="panel onboarding onboarding-panel-3 hide" id="onboarding-panel-3">
+    <img class="onboarding-img" alt="How Containers Work" src="/img/onboarding-3.png" />
+    <h3 class="onboarding-title">A place for everything, and everything in its place.</h3>
```
Comment (pdehaan) [inline webextension/popup.html]: minor nit, but I haven't seen a reference UI spec for this, but should the `<h3>` tags have trailing periods or no?
I notice the `onboarding-panel-1 > h3` doesn't, but `onboarding-panel-2` and `onboarding-panel-3` do.

**Comment by pdehaan** [comment]
Also, I think we may need to optimize the 3 new onboarding-*.png images (possibly others too).

Currently:
```sh
$ ls -lh webextension/img/onboarding-*.png
    27K Jan 30 14:20 webextension/img/onboarding-1.png
    28K Jan 30 14:20 webextension/img/onboarding-2.png
    22K Jan 30 14:20 webextension/img/onboarding-3.png
```

But if I optimize w/ [**imagemin-cli**](http://npm.im/imagemin-cli) (via `$(npm bin)/imagemin webextension/img/*.png --out-dir=optimized`), those numbers drop considerably (27K -> 11K):

```sh
$ ls -lh optimized/onboarding-*.png
    11K Jan 30 14:44 dumpy/onboarding-1.png
    10K Jan 30 14:44 dumpy/onboarding-2.png
   7.3K Jan 30 14:44 dumpy/onboarding-3.png
```

**Comment by groovecoder** [comment]
Optimized all `*.png` images in `webextension/img`

**Comment by groovecoder** [comment]
Not sure about the periods on the onboarding panel titles. I'll leave that for final polish as needed. Merging this to avoid bit-rot.

**Pull request merged**